File: java/test-fixtures/regressed/src/test/java/qg/LibTest.java
package qg;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

class LibTest {
  // REGRESSION test: assertion errada de proposito.
  @Test
  void addFailingOnPurpose() {
    assertEquals(999, Lib.add(2, 3));
  }

  @Test
  void multiplyPositive() {
    assertEquals(12, Lib.multiply(3, 4));
  }
}

File: kotlin/test-fixtures/regressed/src/test/kotlin/qg/LibTest.kt
package qg

import kotlin.test.Test
import kotlin.test.assertEquals

class LibTest {
    // REGRESSION test: assertion wrong on purpose.
    @Test
    fun addFailingOnPurpose() {
        assertEquals(999, add(2, 3))
    }

    @Test
    fun multiplyPositive() {
        assertEquals(12, multiply(3, 4))
    }
}
